modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsPropsTransformer.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {createRenderURL, openSelectionModal} from 'frontend-js-web';
+
+import {ACTIONS} from './actions';
```

Comment:
There is no need to split actions into a separate file. `ACTIONS` array is fine if you like it, but I find in unnecessary complication, unless there are many actions (3+ at least, maybe 4+). To me, a `propsTransformer` like in https://github.com/liferay-echo/liferay-portal/pull/3575 looks cleaner.

Reply:
I usually do that when it's the actions are used by various "propTransformers", but yes in this cases it's not needed ... as for it looking cleaner that's totally opinionated - everyone has their preferences.

Reply:
Changed [here](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/EditPasswordPolicyAssignmentsManagementToolbarPropsTransformer.js#L17-L69)